### PR TITLE
cmd ref: review option defaults and sentence priority per API params

### PR DESCRIPTION
**Pull request by jorgeorpinel**
> Extracted from https://github.com/iterative/dvc.org/pull/908#pullrequestreview-356423629
> and https://github.com/iterative/dvc.org/pull/908#pullrequestreview-356423672

### CODE REVIEW

File: public/static/docs/command-reference/run.md [hunk 1]
```diff
@@ -141,18 +144,14 @@ data pipeline (e.g. random numbers, time functions, hardware dependency, etc.)
...
   confirmation.
 
-- `--ignore-build-cache` - has an effect if an equivalent stage file exists
-  (same dependencies, outputs, and `command` to execute), that has been already
-  executed, and is up to date. In this case, `dvc run` won't normally execute
-  the `command` again. The exception is when the existing stage is considered
-  always changed (see `--always-changed` option). This option gives a way to
-  forcefully execute the `command` anyway. Useful if the command's code is
+- `--ignore-build-cache` - forcefully execute the `command` again, even if an
+  equivalent stage file exists. Useful if the command's code is
```
Comment (shcheklein) [inline public/static/docs/command-reference/run.md]: I would clarify what do we mean by equivalent - same list of same versions of deps, same list of outs, same command. Even if it's part of the different pipeline.

Comment (jorgeorpinel) [reply public/static/docs/command-reference/run.md]: "equivalent stage file" is defined exactly in the description already, so I didn't also add it here, to keep the paragraph not-so-long. Should we repeat it anyway?

Comment (jorgeorpinel) [reply public/static/docs/command-reference/run.md]: > I have an "equivalent" (hehe) question for the api docs PR. Should we repeat default behavior in each option paragraph, even when they are explained in the definition?

Comment (jorgeorpinel) [reply public/static/docs/command-reference/run.md]: p.s. in this case I'm changing the explanation a little to "even if the
  same `dvc run` command has already been run in this workspace"... See 46eb0551

**Pull request merged**